### DISCUSSION TITLE
Auto Pilot Issues

### DISCUSSION THREAD

**Discussion by dashawn11** [flybywiresim/aircraft#4386] [General]
I downloaded stable version, but having issues. Auto pilot keeps disengaging and violently turning to the left. Plane crashes. It will not make normal turns left or right. It turns too fast. I was only able to get one flight done from San Juan to Orlando, FL but auto-pilot act crazy when getting ready to land. Please help. I love to at least finish one good flight. I tried stable version, experimental version etc. Same issues.

The issue always happens at the same way point. As example, on route from KIAD to KATL, the above issue always happens at TEEMS way point.

**Comment by ExampleWasTaken** [comment]
Try increasing your dead zones to 20%

**Comment by Pecheur74** [comment]
I also have AP issues. I really don't understand where it comes from. It seems random. For example, I am in cruise flight, AP engaged, completely straight path. It works fine for some time and then suddenly the plane starts to bank right or left and it can get very brutal if I don't react immediately. It seemed to work better with "selected heading" mode but then during approach phases it sometimes doesn't follow the heading I indicate and tries to go elsewhere. I really don't get it. 
I will try increasing the dead zones.

**Reply by ninoz** [reply]
There is a bug within the flight planner on the mod and the vanilla A320.  When you put in your approach it seems to double back to the previous waypoint, you need to use the DIR TO function to select the next waypoint, I raised an issue here: 

https://github.com/flybywiresim/a32nx/issues/4254
